kaldb/src/main/java/com/slack/kaldb/logstore/schema/SchemaAwareLogDocumentBuilderImpl.java

```diff
@@ -347,6 +360,146 @@ public static SchemaAwareLogDocumentBuilderImpl build(
   }
 
   @Override
+  public Document fromMessage(Trace.Span message) throws JsonProcessingException {
```

Comment:
As an intermediate step this isn't awful, but it's also not my favorite. I know this will be simplified though as we work through this implementation in the followup PRs.